README.md

```diff
@@ -21,12 +21,33 @@ Mongoose models can be found in `models` directory. To add a new model use the s
 
 To connect to mongo
 
+
 ```javascript
-Connector.connect('mongodb://mongoUriGoseHere:27017/db', {})
-  .then(function(db) {
-    // has connection object
+
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var customSchemas = {
+    //The key is the same as the datasetId (e.g. workorders)
+    workorders: function(mongooseConnection) {
+        var customWorkorderSchema = new Schema({
+            ...
+            name: {type: String}
+            ...
+        },  {timestamps: true});
+        
+        mongooseConnection.model('CustomWorkorderModel', customWorkorderSchema);
+    }
+    ... 
+    //Any other custom schemas
+    ...
+};
+
+Connector.connect('mongodb://mongoUriGoseHere:27017/db', {}, customSchemas)
```

Comment:
It maybe worth to mention where we can find it in cloud app. 
https://github.com/wtrocki/raincatcher-demo-cloud/blob/b7564468ff9107f71a75cda48c3aa71b5914e558/lib/storage/mongoose-store.js#L11-L11